pages/404.tsx

```diff
@@ -0,0 +1,54 @@
+import type { NextPage } from 'next'
+import { useContext } from 'react'
+import { ThemeContext } from '../context/ThemeContext'
+import { Banner } from '../components/home/Banner'
+
+interface ErrorProps {
+  status: Number,
+  message: String
+}
+
+const Error: NextPage<ErrorProps> = ({}) => {
+  const { isDarkTheme } = useContext(ThemeContext)
+  return (
+    <div className="bg-[#f5f5f5] dark:bg-[#20201E] relative block">
```

Comment:
all the colors here are defined in tailwind.config.js eg use `bg-lighter dark:bg-dark`